quest/wavefunction.py

```diff
@@ -0,0 +1,42 @@
+"""
+A basic wave function class
+"""
+
+import psi4
+
+
+class WaveFunction(object):
+    """
+    Basic wave function class
+
+    Members:
```

Comment:
`Wavefunction` is a single word so do not add a capitol in the middle. Please see the NumPy style docs [here](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt).